subsys/bluetooth/audio/Kconfig.mcs

```diff
@@ -51,6 +37,38 @@ config BT_MCC
 
 if BT_MCC
 
+config BT_MCC_MEDIA_PLAYER_NAME_MAX
+	int "Max length of media player name"
+	default 20
+	range 1 255
+	help
+	  Sets the maximum number of bytes (including the null termination) of
+	  the name of the media player.
```

Comment:
Perhaps it makes sense to add what happens if the client receives a name larger than this value (truncation or discarding).

Reply:
Also applies the configs below